grails-core/src/main/groovy/org/grails/plugins/AbstractGrailsPluginManager.java

```diff
@@ -315,7 +323,7 @@ public void registerProvidedArtefacts(GrailsApplication app) {
     }
 
     private boolean isAlreadyRegistered(GrailsApplication app, Class<?> artefact, String shortName) {
```

Comment:
The `shortName` can be removed from this method now

Reply:
done